app/lib/frontend/handlers/atom_feed.dart

```diff
@@ -132,13 +134,20 @@ Feed feedFromPackageVersions(Uri requestedUri, List<PackageVersion> versions) {
         activeConfiguration.primarySiteUri.replace(path: pkgPage).toString();
     final alternateTitle = version.package;
 
-    // TODO: use only qualifiedVersion as seed after the domain migration
-    final seed = (requestedUri.host == 'pub.dartlang.org')
-        ? requestedUri
-            .resolve('/packages/${version.package}#${version.version}')
-            .toString()
-        : version.qualifiedVersionKey.toString();
-    final id = uuid.v5(Uuid.NAMESPACE_URL, seed);
+    String id;
+    if (version.created.isAfter(DateTime(2020, 04, 04))) {
+      final hash =
+          sha512.convert(utf8.encode('${version.package}/${version.version}'));
+      id = formatUuid(hash.bytes.sublist(0, 16));
```

Comment:
`hash` is not a `UUID` and formatting as such is not correct.

Reply:
We need to encode UUID version in there, right? It's a hack... IMO we should move `formatUuid` into this file as a private method.. write a comment about how this is not real UUIDs.. because we're abusing notation -- but we don't really care for `feed.atom` right?